HoneyDo.Web/ClientApp/src/components/Tasks/Form.tsx

```diff
@@ -44,47 +64,63 @@ class TaskForm extends Component<TaskFormProps, TaskFormState> {
     }
   }
 
-  handleChange = () => (event: ChangeEvent<HTMLInputElement>) => {
+  handleNameChange = () => (event: ChangeEvent<HTMLInputElement>) => {
     const name = event.target.value;
     this.setState({ name, hasError: false });
   };
 
+  handleDateChange = (dates: Date[]) => {
```

Comment:
Why is this taking an array? It should have `currentDate` and `newDate`?

Reply:
Flatpickr can return multiple dates. Like a date range for instance.

Reply:
Cool. Just making sure.